ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/handlers/HelpCommand.java

```diff
@@ -47,13 +55,17 @@ public void run(String... args) {
             String helpFile = PropertiesLoader.getProperty(HELP_FILE);
             String helpContent = FileUtils.readResource(helpFile);
             ballerinaShell.outputInfo(helpContent);
+
         } else {
+            String topic = String.join(" ", Arrays.copyOfRange(args, 0, args.length));
             try {
-                StringBuilder content = new StringBuilder();
-                helpProvider.getTopic(args, content);
-                ballerinaShell.outputInfo(content.toString());
+                ballerinaShell.outputInfo(DESCRIPTION_PREFIX + NEW_LINE + bbeHelpProvider.getDescription(topic)
+                        .replaceAll(TAGS, EMPTY_STRING));
+                ballerinaShell.outputInfo(URL_PREFIX + URL + topic.replaceAll(" ", "-"));
+
             } catch (HelpProviderException e) {
-                ballerinaShell.outputError(e.getMessage());
+                ballerinaShell.outputError(NEW_LINE + "Can not find topic " + topic  + NEW_LINE + NEW_LINE +
+                        "For available topics please visit " + URL);
```

Comment:
Does this make sense to ask them to refer to the website for topics? 🤔 If they are referring to the website, then no need for them to go there, look up the topic and come back and look it up again in the shell right? 🤔  @mohanvive WDYT?

Reply:
I don't think this is required. We could simply ask to check for a topic by executing topics command. For example: "Cannot find the topic xyz. Please provide a valid topic value. Use /topics command to see all available topics" ?

Reply:
Yeah. That makes more sense than asking the user to visit the BBE web page.

Reply:
Updated in 6902c5cb3c847d22759f0847ba23676a48e75d8d